### ISSUE TITLE
Branch situation

### ISSUE THREAD

**Issue by ioquatix** [glennrp/libpng#189]
Hello.

I don't understand how development works for this repository.

It looks like you are building tar files and then... using those to make commits?

Where is the right place to make commits?

Are you interested in re-structuring the development process?

**Comment by glennrp** [comment]
That is true for the "master" branch,.  Development occurs in the libpng16 and libpn17
branches,.  Most pull requests are against libpn16,

**Comment by ioquatix** [comment]
What is the difference between libpng16 and libpng17?

**Comment by glennrp** [comment]
libpng16 is the "released" branch and libpng17 is a "development" branch,.

libpng17 hasn't had much attention for the past 6 months or so,.

Glenn

**Comment by ioquatix** [comment]
I understand that, but what are the goals of libpng17?

**Comment by jbowler** [comment]
@ctruta - issue is a misunderstanding (as explained elsewhere) and it's a question, not an issue, which is appropriate to a discussion.  I suggest you close, or move it to a discussion.
